docs/en/Remote-Logging.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: Remote Logging
+group: Features
+priority: 5
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+## Configure Alluxio
+Suppose the hostname of the log server is AlluxioLogServer, and the port is 45010.
+Refer to [Running Alluxio on a cluster](Running-Alluxio-on-a-Cluster.html) for instructions in deploying
+Alluxio in a cluster.
+
+By default, remote logging is not enabled. To enable Alluxio remote logging, you can either set a
+few environment variables or modify the JVM properties directly.
+
+This section describes these two different approaches step by step. There is no requirement on where
+the log server must run, as long as the other Alluxio servers have access to it. In our example, we
+run the log server on the same machine as the primary master. If you want to run the log server on
```

Comment:
there isn't a designated primary master; the master which is primary could change at any time. We can only say "on the same machine as a master"

Reply:
Done.